tests/ui/lifetimes/any-lifetime-escape-higher-rank.rs

```diff
@@ -1,11 +1,15 @@
+//! Checks that `std::any::Any` cannot be used to circumvent lifetime rules
+//! particularly with higher-rank types.
```

Comment:
```suggestion
//! with higher-rank types.
```

I think this test is specifically about HKTs